ansible/templates/cluster-template.yaml.j2

```diff
@@ -700,43 +703,46 @@ worker:
+          ExecStartPre=-/bin/bash -c 'docker kill "$(docker ps -q --filter=name=%p-s3_)" > /dev/null 2>&1'
           ExecStartPre=-/bin/bash -c 'docker rm "$(docker ps -q --filter=name=%p-filter_)" > /dev/null 2>&1'
-          ExecStartPre=-/bin/bash -c 'docker rm "$(docker ps -q --filter=name=%p-http_)" > /dev/null 2>&1'
-          ExecStartPre=/bin/bash -c 'docker history coco/coco-splunk-http-forwarder:$DOCKER_FORWARDER_VERSION >/dev/null 2>&1 \
-            || docker pull coco/coco-splunk-http-forwarder:$DOCKER_FORWARDER_VERSION'
+          ExecStartPre=-/bin/bash -c 'docker rm "$(docker ps -q --filter=name=%p-s3_)" > /dev/null 2>&1'
+          ExecStartPre=/bin/bash -c 'docker history coco/log-collector:$DOCKER_LOG_COLLECTOR_VERSION >/dev/null 2>&1 \
+            || docker pull coco/log-collector:$DOCKER_LOG_COLLECTOR_VERSION'
           ExecStartPre=/bin/bash -c 'docker history coco/coco-logfilter:$DOCKER_LOGFILTER_VERSION >/dev/null 2>&1 \
             || docker pull coco/coco-logfilter:$DOCKER_LOGFILTER_VERSION'
           ExecStart=/bin/sh -c '\
             set -e; \
             alias etcdctl_cmd="/usr/bin/etcdctl --ca-file=/etc/kubernetes/ssl/ca.pem --cert-file=/etc/kubernetes/ssl/etcd-client.pem --key-file=/etc/kubernetes/ssl/etcd-client-key.pem --endpoints=${ETCD_ENDPOINTS}"; \
-            export FORWARD_URL=$(etcdctl_cmd get /ft/config/splunk-forwarder/splunk_hec_url); \
-            export TOKEN=$(etcdctl_cmd get /ft/config/splunk-forwarder/splunk_hec_token); \
             BATCHSIZE=$(etcdctl_cmd get /ft/config/splunk-forwarder/batchsize  2>/dev/null) || BATCHSIZE=10; \
             export ENV=$(etcdctl_cmd get /ft/config/environment_tag); \
+            export DNS_ADDRESS=$(etcdctl_cmd get /ft/config/public_address); \
+            export BUCKET_NAME=$(etcdctl_cmd get /ft/config/splunk-forwarder/bucket_name); \
+            export AWS_REGION=$(etcdctl_cmd get /ft/config/splunk-forwarder/aws_region); \
+            export AWS_ACCESS_KEY_ID=$(etcdctl_cmd get /ft/config/splunk-forwarder/aws_access_key_id); \
+            export AWS_SECRET_ACCESS_KEY=$(etcdctl_cmd get /ft/config/splunk-forwarder/aws_secret_access_key); \
             journalctl -a -f --since=now --output=json \
-            | docker run -i --log-driver=none -e=ENV=$ENV --rm --name %p-filter_$(uuidgen) --memory="256m" coco/coco-logfilter:$DOCKER_LOGFILTER_VERSION \
-            | docker run -i --log-driver=none --rm --name %p-http_$(uuidgen) --memory="256m" -e="FORWARD_URL=$FORWARD_URL" -e="ENV=$ENV" -e="HOSTNAME=$(hostname -s)" -e="WORKERS=8" -e="BUFFER=256" -e="TOKEN=$TOKEN" -e="BATCHSIZE=$BATCHSIZE" -e="BATCHTIMER=5" coco/coco-splunk-http-forwarder:$DOCKER_FORWARDER_VERSION'
+            | docker run -i --log-driver=none -e=ENV=$ENV -e="DNS_ADDRESS=$DNS_ADDRESS" --rm --name %p-filter_$(uuidgen) --memory="256m" coco/coco-logfilter:$DOCKER_LOGFILTER_VERSION \
```

Comment:
What does `DNS_ADDRESS` refer to?

Reply:
The public dns name of the cluster - without the region level. 

It is used to deduce whether the cluster is actively serving responses, by verifying if the environment name can be found among the cnames for the original dns name. 

For example, if the dns_address is `staging-delivery.ft.com`, this evaluates to a `staging-eu-delivery.ft.com` cname, which has the `staging-eu` environment, so it serves traffic.